File: Modules/NodeJS/Day12/Smit vadhiya/Assignment/mongoDb/empCollection/employees.js
const mongoose = require('mongoose')


const employeeSchema = mongoose.Schema({
    CitizenshipId: {type : Number, required : true} ,
    AddressLine1: {type : String, required : true},
    AddressLine2: String,
    AddressLine3: String,
    assignments: [ Number ],
    CitizenshipLegislationCode: String,
    CitizenshipStatus: String,
    CitizenshipToDate: Date,
    City: String,
    CorrespondenceLanguage: String,
    Country: String,
    CreationDate: Date,
    DateOfBirth: Date,
    directReports: [ Number ],
    DisplayName: String,
    DriversLicenseExpirationDate: Date,
    DriversLicenseId: Number ,
    DriversLicenseIssuingCountry: String,
    EffectiveStartDate: Date,
    Ethnicity: String,
    FirstName: String,
    Gender: String,
    HireDate: Date,
    HomeFaxAreaCode: String,
    HomeFaxCountryCode: String,
    HomeFaxExtension: String,
    HomeFaxLegislationCode: String,
    HomeFaxNumber: String,
    HomePhoneAreaCode: String,
    HomePhoneCountryCode: String,
    HomePhoneExtension: String,
    HomePhoneLegislationCode: String,
    HomePhoneNumber: String,
    Honors: String,
    LastName: String,
    LastUpdateDate: Date,
    LegalEntityId: Number ,
    LicenseNumber: String,
    links: [ Number ],
    MaritalStatus: String,
    MiddleName: String,
    MilitaryVetStatus: String,
    NameSuffix: String,
    NationalId: String,
    NationalIdCountry: String
})

const Employees = mongoose.model('Employees',employeeSchema)

module.exports = Employees

File: Modules/C#/Day15/Palak Patel/Assignment/Assignment/Assignment/Models/employees.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;


namespace Assignment.Models
{
    public class employees
    {
        public int id { get; set; }
        public string Address { get; set; }
        public int? CitizenshipId { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public string DisplayName { get; set; }
        public int? DrivingLicenseId { get; set; }
        public string FirstName { get; set; }
        public string Gender { get; set; }
        public DateTime? HireDate { get; set; }
        public string LastName { get; set; }
        public string LicenseNumber { get; set; }
        public string MaritalStatus { get; set; }
        public string MiddleName { get; set; }
        public string NationalId { get; set; }
        public IEnumerable<EmpAssigned> Assignments { get; set; }
    }
}
